app/extractors/saarland_pdf_extractor.rb

```diff
@@ -0,0 +1,29 @@
+class SaarlandPDFExtractor
+  def initialize(paper)
+    @contents = paper.contents
+  end
+
+  ORIGINATORS = /Anfrage\s+de[rs]\s+Abgeordneten.*\n/m
+
+  def extract_originators
```

Comment:
Assignment Branch Condition size for extract_originators is too high. [18.55/15]